Formula/u/utiluti.rb

```diff
@@ -0,0 +1,42 @@
+class Utiluti < Formula
+  desc "Command-line utility to work with default apps"
+  homepage "https://github.com/scriptingosx/utiluti"
+  url "https://github.com/scriptingosx/utiluti/archive/refs/tags/v1.3.tar.gz"
+  sha256 "98d72888421cf048d157da3b64dcc4bdc165c4899de987636a1cac022e5bd7ff"
+  license "Apache-2.0"
+
+  depends_on :macos
+  depends_on macos: :big_sur
+
+  uses_from_macos "swift" => :build, since: :sonoma # swift 6.0+
+
+  patch :DATA
+
+  def install
+    system "swift", "build", "--disable-sandbox", "--configuration", "release"
+    system "swift", "package", "--disable-sandbox", "plugin", "generate-manual"
+
+    release_dir = buildpath/".build/release"
+    manual_dir = buildpath/".build/plugins/GenerateManual/outputs/utiluti"
+
+    bin.install release_dir/"utiluti"
+    man1.install manual_dir/"utiluti.1"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/utiluti --version")
```

Comment:
need a more substantial functional check?

Reply:
@chenrui333 thanks, I missed this before submission. Fixed in 4cbf440.